buf/registry/module/v1beta1/label_service.proto

```diff
@@ -170,7 +170,8 @@ message CreateOrUpdateLabelsRequest {
     LabelRef label_ref = 1 [(buf.validate.field).required = true];
     // The id of the Commit to associate with the Label.
     //
-    // If the Label already existed, the Label will now point to this Commit.
+    // If the Label already existed, the Label will now point to this Commit, as long as this Commit
+    // is newer than the Commit that the Label is currently pointing to.
```

Comment:
I think we should be explicit that it is an error to attempt to update a label to point to an older commit.

```suggestion
    // If the Label already existed, the Label will now point to this Commit as long as this Commit
    // is newer than the Commit that the Label is currently pointing to, otherwise an error is returned.
```